weldx/asdf/schemas/weldx.bam.de/weldx/core/transformations/local_coordinate_system-1.0.0.yaml

```diff
@@ -105,15 +105,18 @@ properties:
   coordinates:
     description: |
       An ndarray containing the coordinates.
-    tag: "tag:weldx.bam.de:weldx/core/variable-1.0.0"
+
+    oneOf:
+      - tag: "tag:weldx.bam.de:weldx/core/variable-1.0.0"
+      - tag: "tag:weldx.bam.de:weldx/core/time_series-1.0.0"
   orientations:
     description: |
       An ndarray containing the orientations.
     tag: "tag:weldx.bam.de:weldx/core/variable-1.0.0"
 
 wx_shape:
-  (coordinates):
-    data: [(t), 3]
+  # (coordinates):
+    # data: [(t), 3]
```

Comment:
We need to decide if and how we want to use shape validation if the coordinates are a `TimeSeries`

Reply:
cref #86